### PR TITLE
chore: bump docker-compose deps

### PR DESCRIPTION
**Pull request by enitrat**
Time spent on this PR:

Resolves: #<Issue number>

# Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Testing

# What is the new behavior?

# Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by greged93** [comment]
did you release on Kakarot repo?

**Comment by enitrat** [comment]
waiting for approval @greged93 @tcoratger

**Comment by Eikix** [comment]
Still fails

**Comment by greged93** [comment]
need to update the Katana genesis in order to include pre-eip 155 transaction hashes I think.

File: Makefile
Comment (tcoratger) [inline Makefile:46]: Even `v1.0.0-alpha.1` is ok no? since cairo 2.7 is in `v1.0.0-alpha.2`

Comment (greged93) [reply Makefile:46]: I guess it's ok, I just went with the same version as on the Kakarot repo

**Review by tcoratger — APPROVED** [review]

**Pull request merged**
